GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -2744,7 +2741,7 @@ private void toolStripButtonPull_DropDownOpened(object sender, EventArgs e)
 
         private void FormBrowse_Activated(object sender, EventArgs e)
         {
-            this.InvokeAsync(OnActivate);
+            this.InvokeAsyncDoNotUseInNewCode(OnActivate);
```

Comment:
These are locations that haven't yet converted to use `FileAndForget` because I haven't evaluated the case in detail to ensure the original error reporting semantics are preserved.